### ISSUE TITLE
iframe height (and styles) are reseted everytime

### ISSUE THREAD

**Issue by yairEO** [tiff/wysihtml5#277]
if i'm trying to change the iframe height in anyway, when it gets focused again, all the styles are re-seted back. where in the code can I cancel this unwanted thing?

**Comment by zrod** [comment]
Would setting a min-height property work? (I'm supposing you're trying to setup the resize plugin mentioned on issue 134 of bootstrap-wysihtml5)

**Comment by yairEO** [comment]
yes I am and I don't understand why in the code the styles are getting reset.
it makes no sense, and if you could just point me to the line that does that I could fix this issue.

min-height does help, but generally this is not a good thing in the plugin, to force itself upon the poor iframe element
